modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+
+require_dependency 'va_facilities/application_controller'
+require_dependency 'va_facilities/geo_serializer'
+require_dependency 'va_facilities/csv_serializer'
+
+module VaFacilities
+  module V0
+    class FacilitiesController < ApplicationController
+      include ActionController::MimeResponds
+      skip_before_action(:authenticate)
+      before_action :set_default_format
+      before_action :validate_params, only: [:index]
+
+      TYPE_SERVICE_ERR = 'Filtering by services is not allowed unless a facility type is specified'
+
+      def all
+        resource = BaseFacility.all.order(:unique_id)
```

Comment:
Minor concerns about limits on this query

Reply:
Yes...we intend to cache the full result set to a file and serve it from there rather than hitting the db every time. But trying to get this going before Monday. It's protected by api tokens and we can institute rate limiting if somebody is hitting it hard.